package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cuttle",
-  "version": "3.3.1",
+  "version": "3.3.2",
```

Let's bump this to 3.3.5
```suggestion
  "version": "3.3.5",
```

---

This change was made.